src/video/wayland/SDL_waylanddyn.h

```diff
@@ -100,6 +114,30 @@ void SDL_WAYLAND_UnloadSymbols(void);
 #define wl_data_source_interface (*WAYLAND_wl_data_source_interface)
 #define wl_data_device_manager_interface (*WAYLAND_wl_data_device_manager_interface)
 
+#ifdef HAVE_LIBDECOR_H
```

Comment:
This was the only issue I was able to find in the build process as of the latest revision - this resolves a linker error where the libdecor includes that happen later get ripped up by these defines and cause double definitions. With this I'm able to build libdecor+SDL and run CSD successfully, albeit with some unrelated issues (will try and write up everything on Monday).

```suggestion
#ifdef HAVE_LIBDECOR_H
/* Must be included before our defines */
#include <libdecor.h>

```

Reply:
Is this a linking error on your local system? I didn't see it on my configuration or the CI.

Reply:
Yes, this was on my Wayland test box - Fedora 33 Workstation, libdecoration installed to `/usr/local/`, with `dnf builddep SDL2`.